Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM maven:3-jdk-8-onbuild
+
+ENV JVM_GC_OPTS="-XX:+UseG1GC -XX:MaxGCPauseMillis=200 -XX:+UseStringDeduplication -XX:+ParallelRefProcEnabled"
```

Comment:
The idea behind putting an entry point is to make sure that we are able to put some additional Java opts.

Does that make sense? I used the pattern from the appengine-openjdk runtime docker image.